apps/jest-expo-mock-generator/App.js

```diff
@@ -16,11 +23,59 @@ if (!ExpoNativeModuleIntrospection) {
   );
 }
 
+const keysOrder = ['type', 'functionType', 'name', 'argumentsCount', 'key'];
+
+function isNumeric(str) {
+  if (typeof str != 'string') return false; // we only process strings!
+  return (
+    !isNaN(str) && // use type coercion to parse the _entirety_ of the string (`parseFloat` alone does not do this)...
+    !isNaN(parseFloat(str))
+  ); // ...and ensure strings of whitespace fail
+}
+
+const replacer = (_key, value) => {
+  if (value instanceof Object && !(value instanceof Array)) {
+    return Object.keys(value)
+      .sort(function (a, b) {
+        if (keysOrder.indexOf(a) !== -1 || keysOrder.indexOf(b) !== -1) {
+          return (
+            (keysOrder.includes(a) ? keysOrder.indexOf(a) : Infinity) -
+            (keysOrder.includes(b) ? keysOrder.indexOf(b) : Infinity)
+          );
+        } else {
+          return a.localeCompare(b);
+        }
+      })
+      .reduce((sorted, key) => {
+        sorted[key] = value[key];
+        return sorted;
+      }, {});
+  }
+  if (value instanceof Array) {
+    // sorts by numeric keys eg. { name: 'isAvailableAsync', argumentsCount: 0, key: 0 },
+    if (value?.[0]?.key && isNumeric(value?.[0]?.key)) {
+      return value.sort((a, b) => Number(a?.key) > Number(b?.key));
+    }
+    // sorts by string keys  eg. { name: 'getNetworkStateAsync', argumentsCount: 0, key: 'getNetworkStateAsync' },
+    if (value?.[0]?.key) {
+      return value.sort((a, b) => a?.key?.localeCompare?.(b?.key));
+    }
+    // sort other arrays
+    return value?.sort((a, b) => a?.localeCompare?.(b)) ?? value;
+  }
+  return value;
+};
+
 export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
```

Comment:
Perhaps can be replaced with
```suggestion
  state = {};

```